### PR TITLE
more steps along the way to getting new robots actually running

### PR DESCRIPTION
**Pull request by ndushay**
It seems I am still tweaking to get robots running properly.  :-(

We can decide to merge this and then do add on PRs later.

### CODE REVIEW

File: config/boot.rb [hunk 1]
```diff
@@ -35,15 +35,15 @@
...
-#   end
-# end
+begin
+  if defined? REDIS_TIMEOUT
+    _server, _namespace = REDIS_URL.split('/', 2)
+    _host, _port, _db = _server.split(':')
+    _redis = Redis.new(host: _host, port: _port, thread_safe: true, db: _db, timeout: REDIS_TIMEOUT.to_f)
+    Resque.redis = Redis::Namespace.new(_namespace, redis: _redis)
+  else
+    Resque.redis = REDIS_URL
```
Comment (atz) [inline config/boot.rb]: Agree this `else` was always useless.  Not sure about the namespace and `REDIS_TIMEOUT` though.  Are we sure those are disposable?

Comment (ndushay) [reply config/boot.rb]: Hmm - I just copied boilerplate;  I just renamed all vars to NOT start with underscore as they are all used.

File: config/boot.rb [hunk 2]
```diff
@@ -35,15 +35,15 @@
...
-#   end
-# end
+begin
+  if defined? REDIS_TIMEOUT
+    server, namespace = REDIS_URL.split('/', 2)
+    host, port, db = server.split(':')
+    redis = Redis.new(host: host, port: port, thread_safe: true, db: db, timeout: REDIS_TIMEOUT.to_f)
+    Resque.redis = Redis::Namespace.new(namespace, redis: redis)
+  else
+    Resque.redis = REDIS_URL
```
Comment (atz) [inline config/boot.rb]: Same feedback.  This `else` clause shouldn't exist.  It should just rely on line 37's assignment, which is smarter and better.  Alternatively, line 37 should be deleted, but it seems to be the preferable version.

Comment (ndushay) [reply config/boot.rb]: gotcha.  Removed in commit below.

**Pull request merged**